jsonrpc/src/schemas/portal.json

```diff
@@ -38,5 +38,24 @@
     "title": "UDP port number",
     "type": "string",
     "pattern": "^([1-9][0-9]{0,3}|[1-5][0-9]{4}|6[0-4][0-9]{3}|65[0-4][0-9]{2}|655[0-2][0-9]|6553[0-5])$"
+  },
+  "ContentItem": {
+    "title": "content_item",
+    "type": "array",
```

Comment:
Why is this an `array`? Shouldn't it be an `object`?

Reply:
@ScottyPoi said it should be a tupel? but that doesn't exist so I did an array.

I will let you guys discuss what we should use as I am impartial as long as I can use this RPC command to test the spec. I do think an object would be potentially cleaner though, but I haven't looked too into this yet

Reply:
i'm OK with it either way

Reply:
I implemented it with tuples and it feels good so I am on the tupel side now

Reply:
> and when they are translated from other languages I think they are typically translated to objects.

sorry for the confusion.  i thought "tuple" was universally an array-like structure, but with constraints (i.e. length: 2)